.github/workflows/build-test-deploy.yml

```diff
@@ -778,6 +778,25 @@ jobs:
           command: nix -- build .#price-feed
           token: "${{ secrets.CACHIX_AUTH_TOKEN }}"
 
+  package-dali-metadata:
+    needs: common-deps
+    runs-on:
+      - self-hosted
+      - linux
+      - x64
+      - sre
+    concurrency:
+      group: ${{ github.workflow }}-package-dali-metadata-${{ github.ref }}
+      cancel-in-progress: true
+    container:
+      image: niteo/nixpkgs-nixos-22.05:316b762afdb9e142a803f29c49a88b4a47db80ee
+    steps:
+      - uses: actions/checkout@v3
+      - uses: "./.github/templates/watch-exec"
+        with:
+          command: nix -- build .#subwasm-get-dali-metadata
```

Comment:
should we build for picasso and composable too?

Reply:
I think it will be useful. Let's add it.

Reply:
this one was next task in cu. we tried to end to end floq for one client and then wnable generation for all.

Reply:
dali is prio for bridge.

Reply:
and it is not clear should we buld historical clienta or not.

Reply:
this doesn't require a lot of places, JSON with metadata is less than 1.5 Mb.
I added a packaging of picasso and composable too.